src/v/kafka/protocol/CMakeLists.txt

```diff
@@ -3,8 +3,6 @@ add_subdirectory(schemata)
 v_cc_library(
   NAME kafka_protocol
   SRCS
-    errors.cc
-  DEPS
-    v::kafka_request_schemata)
+    errors.cc)
```

Comment:
what do you think about replacing `v::kafka_request_schemata` and `v::kafka_protocol` with just `v::kafka_protocol`? early on it wasn't clear how these libraries might grow independently. we are far enough along now that it doesn't seem like they are going to grow independently.

Reply:
yeah, that'd be better. i just updated this change to merge `v::kafka_request_schemata` into `v::kafka_protocol`. 

would you mind taking another look?